src/panels/Menu/TopMenuBar/Menus/FileMenu.tsx

```diff
@@ -0,0 +1,48 @@
+import { Kbd, Menu, Text } from '@mantine/core';
+import { modals } from '@mantine/modals';
+
+import { useGetBoolPropertyValue, useOpenSpaceApi } from '@/api/hooks';
+import { AddFileIcon, ConsoleIcon, ExitAppIcon } from '@/icons/icons';
+
+import { TopBarMenuWrapper } from '../TopBarMenuWrapper';
+
+export function FileMenu() {
+  const luaApi = useOpenSpaceApi();
+
+  const [isConsoleVisible, setIsConsoleVisible] =
+    useGetBoolPropertyValue('LuaConsole.IsVisible');
+
+  function toggleLuaConsole() {
+    setIsConsoleVisible(!isConsoleVisible);
+  }
+
+  function toggleShutdown() {
+    return modals.openConfirmModal({
+      title: 'Confirm action',
+      children: <Text>Are you sure you want to quit OpenSpace?</Text>,
+      labels: { confirm: 'Quit', cancel: 'Cancel' },
+      confirmProps: { color: 'red' },
```

Comment:
This button is no longer red (due to default button style(?)) add variant="filled"?